Client/game_sa/CSettingsSA.cpp

```diff
@@ -590,6 +591,36 @@ void CSettingsSA::GetVehiclesLODDistance(float& fVehiclesLODDistance, float& fTr
     fTrainsPlanesLODDistance = ms_fTrainPlaneLODDistance;
 }
 
+////////////////////////////////////////////////
+//
+// Peds LOD draw distance
+//
+////////////////////////////////////////////////
+ 
+void CSettingsSA::SetPedsLODDistance(float fPedsLODDistance)
+{
+    ms_fPedsLODDistance = fPedsLODDistance;
+}
+ 
+void CSettingsSA::GetPedsLODDistance(float& fPedsLODDistance)
+{
+    fPedsLODDistance = ms_fPedsLODDistance;
+}
+ 
+void CSettingsSA::ResetPedsLODDistance()
+{
+    bool bHighDetailPeds;
+    g_pCore->GetCVars()->Get("high_detail_vehicles", bHighDetailPeds);

```

Comment:
Can it have its own property ? or is it overkilled ?

Reply:
@CrosRoad95 the name clearly says "high_detail_vehicles", any reason for not creating a new property?

Reply:
i'm lazy

Reply:
I can't merge it like this. You can add the new property under high_detail_vehicles property in code. 
```Client\core\CClientVariables.cpp: line 328
Client\core\CSettings.cpp: line 1455, 1680, and 3199
Client\sdk\core\CRenderItemManagerInterface.h: 128
Client\core\Graphics\CRenderItemManager.cpp: line 767
Client\game_sa\CSettingsSA.cpp: line 574 and 613
Client\mods\deathmatch\logic\luadefs\CLuaDrawingDefs.cpp: line 1299
```